packages/next/server/web/spec-extension/cookies.ts

```diff
@@ -59,10 +59,17 @@ export class NextCookies extends Cookies {
     super(response.headers.get('cookie'))
     this.response = response
   }
+  get = (...args: Parameters<Cookies['get']>) => {
```

Comment:
isn't this a recursive type definition? `Cookies["get"]` is referencing `Cookies["get"]`

Reply:
can we be explicit on types here?